prepare

```diff
@@ -35,6 +36,13 @@ sudo DEBIAN_FRONTEND=noninteractive apt install -y --no-install-recommends sbuil
 sudo DEBIAN_FRONTEND=noninteractive apt install -y --no-install-recommends sbuild mmdebstrap distro-info debian-archive-keyring ccache uidmap vcstool python3-rosdep2 colcon python3-colcon-package-information python3-colcon-ros python3-bloom curl apt-cacher-ng || \
 sudo DEBIAN_FRONTEND=noninteractive apt install -y --no-install-recommends sbuild mmdebstrap distro-info debian-archive-keyring ccache uidmap python3-vcstool python3-rosdep python3-colcon-cmake python3-colcon-installed-package-information python3-colcon-library-path python3-colcon-package-information python3-colcon-pkg-config python3-colcon-recursive-crawl python3-colcon-test-result python3-bloom curl apt-cacher-ng
 
+# Set up git-lfs
+sudo mkdir -p /etc/apt/keyrings
```

Comment:
any reason you don't use the git-lfs package in jammy?

Reply:
Done in 2bbca4437a68976f6edb424100776d4a456bdef8